README.md

```diff
@@ -223,7 +219,16 @@ The server will respond with JSON object.
 - The applicationId returned can be used to view the candidate profile in Lever at the url: `https://hire.lever.co/search/application/{applicationId}`. Note that only users logged in to Lever will be able to access that page.
 - On error, we'll send the appropriate HTTP error code and a body of `{ok:false, error:<error string>}`.
 
+### WARNING: POST Application Rate limit
```

Comment:
I recommend moving this warning to the top of the section (line 180). Then include the full detail where it is.